src/libraries/System.Private.CoreLib/src/System/DateTime.cs

```diff
@@ -61,13 +61,15 @@ namespace System
           ISubtractionOperators<DateTime, DateTime, TimeSpan>
     {
         // Number of 100ns ticks per time unit
-        private const long TicksPerMillisecond = 10000;
+        private const long TicksPerMicrosecond = 10;
+        private const long TicksPerMillisecond = TicksPerMicrosecond * 1000;
```

Comment:
nit: Would be better if we move the definition of MicrosecondsPerMillisecond up here and make:

private const long TicksPerMillisecond = TicksPerMicrosecond * MicrosecondsPerMillisecond;